nspawn/nspawn.go

```diff
@@ -174,6 +175,9 @@ func (c *MachineConfig) ConfigArray() ([]string, error) {
 	if len(c.Command) > 0 {
 		args = append(args, c.Command...)
 	}
+	if len(c.Capability) > 0 {
```

Comment:
Can you please put this in front of this block:
```
	if len(c.Command) > 0 {
		args = append(args, c.Command...)
	}
```
Otherwise the added capabilities will be evaluated as additional arguments to a passed command by `systemd-nspawn`.